### PR TITLE
DELTA-4: Verify all the links available on the homepage

### PR DESCRIPTION
**Pull request by sourabhsrk17**

### CODE REVIEW

File: src/main/java/pages/HomePage.java [hunk 1]
```diff
@@ -30,6 +30,15 @@
     @FindBy(xpath = "//div[@class='product-grid home-page-product-grid']/div[2]/div")
     private List<WebElement> listItemsFeature;
 
+    @FindBy(xpath = "//div[@class='footer-block information']/ul/li")
+    private List<WebElement> footerInformationLinks;
```
Comment (shriniwasalle) [inline src/main/java/pages/HomePage.java:34]: No need to create separate xpath for diff footer sections, please create single xpath and test method to verify all the links available on HomePage

Comment (sourabhsrk17) [reply src/main/java/pages/HomePage.java]: done with changes and pushed code

Comment (sourabhsrk17) [reply src/main/java/pages/HomePage.java]: > No need to create separate xpath for diff footer sections, please create single xpath and test method to verify all the links available on HomePage

done with changes and pushed code

**Review by shriniwasalle — APPROVED** [review]
Looks good

**Pull request merged**